wdl/model/draft2/src/test/scala/wdl/types/WomArrayTypeSpec.scala

```diff
@@ -13,7 +13,7 @@ class WomArrayTypeSpec extends FlatSpec with Matchers  {
   behavior of "WomArrayType"
 
   List(WomStringType, WomArrayType(WomIntegerType), WomPairType(WomIntegerType, WomPairType(WomIntegerType, WomIntegerType)), WomOptionalType(WomStringType)) foreach { desiredMemberType =>
-    it should s"be able to construct an empty Array[${desiredMemberType.toDisplayString}] value" in {
+    it should s"be able to construct an empty Array[${desiredMemberType.callCachingName}] value" in {
```

Comment:
1. Agreed, I updated to `friendlyName` and `stableName` which I think are both objectively better and in line with your preference
2. The short answer is, because the "unfriendly" name was all that existed at the time it was written. We can opportunistically introduce the friendly name where it's appropriate, but it's hard to find all of those places in one go